### DISCUSSION TITLE
Installation on Raspberry Pi 4

### DISCUSSION THREAD

**Discussion by kyleti** [rhasspy/piper#542] [General]
I'm having a terrible time getting it installed on my RasPi 4. 
pip install piper-tts 
results in the Error: Cannot install piper-tts==1.1.0 and piper-tts==1.2.0 because these package versions have conflicting dependencies. 
I  tried:  
pip install piper-tts -no-deps 
but then it is missing the onnxruntime. "pip install onnxruntime" gives Error: No matching distribution found for onnxruntime.

**Comment by vyastreb** [comment]
You probably need to use an older (newer) version of pip. I had the same problem with pip3.12.2, but downgrading to pip3.10 worked.
